barcode_detection/localization/iyyun_method/docker/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM ubuntu:16.04
+
+RUN apt-get update
+RUN apt-get install -y cmake git libgtk2.0-dev pkg-config libavcodec-dev \
+libavformat-dev libswscale-dev python-dev python-numpy libtbb2 libtbb-dev \
+libjpeg-dev libpng-dev libtiff-dev libjasper-dev libdc1394-22-dev unzip
+
+RUN apt-get install -y wget
+
+RUN cd \
```

Comment:
`cd` is not necessary

Reply:
I didn't want to specify absolute paths to all directories within one RUN command